baseapp-core/baseapp_core/tests/factories.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import AbstractBaseUser
+
+import factory
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    email = factory.Faker("email")
+    password = factory.PostGenerationMethodCall("set_password", "default")
+    first_name = factory.Faker("first_name")
+    last_name = factory.Faker("last_name")
+
+    class Meta:
+        model = AbstractBaseUser
```

Comment:
let's see if making it dynamic will work:

```
    class Meta:
        model = settings.AUTH_USER_MODEL
```

or by using `from django.contrib.auth import get_user_model`